src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -329,6 +339,20 @@ private static LibraryDependency ToPackageLibraryDependency(IVsReferenceItem ite
             return dependency;
         }
 
+        private static DownloadDependency ToPackageDownloadDependency(IVsReferenceItem item)
+        {
+            var id = item.Name;
+            var versionRange = GetVersionRange(item);
+            if (!(versionRange.HasLowerAndUpperBounds && versionRange.MinVersion.Equals(versionRange.MaxVersion)))
+            {
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, "Strings.Error_PackageDownload_OnlyExactVersionsAreAllowed TODO NK", versionRange.OriginalString));
```

Comment:
Can you resolve this resource string business (`TODO NK`)?

Reply:
fixed